azure/template.json

```diff
@@ -46,12 +68,143 @@
             "tags": "[parameters('tags')]",
             "properties": {
             }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-06-01",
+            "name": "app-service-certificate",
+            "resourceGroup": "[variables('resourceGroupName')]",
```

Comment:
the certificate should be created in the shared environment resource group, the same group the app service plan is in